docs/setup/independent/create-cluster-kubeadm.md

```diff
@@ -179,6 +179,8 @@ as root:
 
   kubeadm join --token <token> <master-ip>:<master-port> --discovery-token-ca-cert-hash sha256:<hash>
 ```
+**Note:** you must save `kubeadm join --token <token> <master-ip>:<master-port> --discovery-token-ca-cert-hash sha256:<hash>`in your local, it's so important once you need join machine to cluster.
+
```

Comment:
Suggestion:

You must save the last line from the output somewhere in your local environment.
It will be needed when you are joining more machines into the cluster.


However, I'm not a native speaker. As I previously commented, I was waiting for some native speakers to help polish this sentence.

Reply:
Thanks